cluster/addons/fluentd-gcp/fluentd-gcp-image/google-fluentd-journal.conf

```diff
@@ -63,7 +63,17 @@
 <match reform.**>
   type record_reformer
   enable_ruby true
-  tag kubernetes.${tag_suffix[4].split('-')[0..-2].join('-')}
+  tag raw.kubernetes.${tag_suffix[4].split('-')[0..-2].join('-')}
+</match>
+
+# Detect exceptions in the log output and forward them as one log entry.
+<match raw.kubernetes.**>
+  type detect_exceptions
+  remove_tag_prefix raw
+  message log
+  multiline_flush_interval 5
+  max_bytes 50000
```

Comment:
I just didn't dare to exhaust the complete limit. Even with a lower limit, large log messages will still be forwarded. It's just that they will not be buffered and exceptions stacks may be separated (as they currently are). Anyway, I've changed it now, since I am not aware of a concrete problem that this could cause. My initial setting was rather 'fear driven'.